JSQMessagesViewController/Factories/JSQMessagesBubbleImageFactory.m

```diff
@@ -72,6 +74,14 @@ - (JSQMessagesBubbleImage *)incomingMessagesBubbleImageWithColor:(UIColor *)colo
     return [self jsq_messagesBubbleImageWithColor:color flippedForIncoming:YES ^ self.isRightToLeftLanguage];
 }
 
+- (BOOL)isRightToLeftLanguage:(UIUserInterfaceLayoutDirection)layoutDirection {
+    if (layoutDirection == UIUserInterfaceLayoutDirectionRightToLeft) {
+        return true;
+    } else {
+    return false;
+    }
+}
```

Comment:
what you wrote here was an entirely new method, not an override for the property.

thus `self.isRightToLeftLanguage` was always `NO`. (which is probably why you had to change the tests) 😅